lib/http_perform.c

```diff
@@ -320,8 +321,6 @@ sf_bool STDCALL http_perform(CURL *curl,
             break;
         }
 
-#ifndef _WIN32
```

Comment:
Just checking  - do we want to enable it for Windows?

Reply:
That's consistent behavior cross drivers I think better to do the same in libsfclient as well
https://github.com/snowflakedb/snowflake-jdbc/blob/master/src/main/java/net/snowflake/client/core/SFBaseSession.java#L564
The application should not set it if they don't want to disable OCSP instead of having the setting ignored in libsfclient specifically on Windows only. This would be a BCR as well.
I think we can't avoid this PR to be a BCR change. Seems supporting OCSP is a BCR for all other drivers.
https://docs.snowflake.com/en/user-guide/ocsp#legacy-client-and-driver-versions